.github/workflows/jib-gradle-prepare-release.yml

```diff
@@ -0,0 +1,63 @@
+name: Prepare Jib Gradle release
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: new release version
+        required: true
+        default: (for example, 0.1.0)
```

Comment:
does this default have some behavior that someone can't accidentally just say "okay" to "(for example, 0.1.0)" ?

Reply:
I can't find anything to that effect. But at least the code checks the version string and exits if not in the right format.
```